apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,109 @@
+- provide a cost benefit: data does not need to be structured at the point of need, it already exists
+- provide a time benefit: data is instantly available, again not requiring computation on the fly
+- act as an always-available fallback in the event of foundation model unavailibilty or disconnection
+
+These features of pre-structured data, combined with foundation models, will unlock and mainstream new technologies and best-practices.
+
+## Better together: pre-structured data & foundation models
+
+With thanks to both foundation models _and_ pre-structured data, we may finally see a renaissance in simulation.
+
+HASH _began life_ building agent-based modeling software. We encountered two main problems in convincing folks to adopt predictive, "generative" AI and simulations in their businesses... _back in 2019._
+
+First, the shape of folks' existing data. Of all the folks we engaged with, generally speaking _none_ had data easily available in nice, neat agent-mappable form. The closest we encountered were enteprises who'd deployed software like Palantir's and developed ["dynamic ontologies"](https://hash.ai/glossary/schemas), but even these rarely represented their entire organization's set of information in a semantic way. In the last year or so, the term "semantic layer" has become vogue term in some corners of Silicon Valley to refer to the ideal of a business whose data is enitrely neatly mapped to real-world "entities", but it remains an ambition for many, as opposed to a realized state of affairs. **We now believe foundation models will help organizations quickly 'type' their data and convert what they have to 'entities' suitable for use in simulations.**
+
+Second, simulations have historically required handoff between:
+
+1. data engineers and analysts who make the data available which is required to instantiate 'agents' in a simulation, and hydrate them with 'properties' that describe them accurately
+2. programmers who're able to encode subject matter expertise _in code_ that provides 'behavioral logic' to the agents, and environment within which they reside
+3. domain experts, who understand how the target system functions, and can actually provide the subject matter expertise and model design required.
+
+These different stakeholders and the communication/coordination/iteration loops involved in getting simulation models accurate _and useful_ were frankly killer. With data structuring and availability problems potentially already solved by foundation models (as outlined above), **we believe foundation models' further potential lies in enabling domain experts to bypass programmers and iterate on simulation logic directly.** Behaviors within simulations are generally relatively simple, standalone things. Natural language behavior developement seems eminently achievable using current-generation (e.g. GPT-4) models.
```

Comment:
> These different stakeholders and the communication/coordination/iteration loops involved in getting simulation models accurate _and useful_ were frankly killer.

This parses as 'These different stakeholders and the communication loops involved were frankly killer' – I _think_ it is aimed at saying 'this often made simulation too difficult', but I think it could be rephrased. Maybe something like this.

```suggestion
The number of different stakeholders and the communication/coordination/iteration loops involved in getting simulation models accurate _and useful_ were often a killer to getting them off the ground. With data structuring and availability problems potentially already solved by foundation models (as outlined above), **we believe foundation models' further potential lies in enabling domain experts to bypass programmers and iterate on simulation logic directly.** Behaviors within simulations are generally relatively simple, standalone things. Natural language behavior developement seems eminently achievable using current-generation (e.g. GPT-4) models.
```